### PR TITLE
Initial commit Freeciv version 2.6.5

### PR DESCRIPTION
**Pull request by jakegoerzen**
This is the Initial commit of Freeciv version 2.6.5

SDL-mixer has been disabled (--disable-sdl-mixer) so no encumbered packages are required.

Dependency tracking had to be disabled (--disable-dependency-tracking) for ./configure to complete successfully.

### CODE REVIEW

File: components/games/freeciv/Makefile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+#
+
+BUILD_BITS= 64
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		freeciv
+COMPONENT_VERSION=	2.6.5
+COMPONENT_SUMMARY=	Free and Open Source empire-building strategy game inspired by the history of human civilization.
+COMPONENT_PROJECT_URL=	https://www.freeciv.org
```
Comment (pyhalov) [inline components/games/freeciv/Makefile]: Minor note, https seems to be not working on the site.

Comment (jakegoerzen) [reply components/games/freeciv/Makefile]: I've changed COMPONENT_PROJECT_URL from https to http, so all should be in order now.

**Comment by AndWac** [comment]
Can you check whether you have installed imagemagick and re-run gmake REQUIRED_PACKAGES?
When I try to publish your package here I run into problems that can be fixed by fixing the dependencies in the Makefile:
When I run gmake REQUIRED_PACKAGES on my build system it will create the following list:
# Auto-generated dependencies
REQUIRED_PACKAGES += compress/bzip2
REQUIRED_PACKAGES += compress/xz
REQUIRED_PACKAGES += image/imagemagick
REQUIRED_PACKAGES += library/desktop/cairo
REQUIRED_PACKAGES += library/desktop/gdk-pixbuf
REQUIRED_PACKAGES += library/desktop/gtk3
REQUIRED_PACKAGES += library/desktop/pango
REQUIRED_PACKAGES += library/glib2
REQUIRED_PACKAGES += library/readline
REQUIRED_PACKAGES += library/zlib
REQUIRED_PACKAGES += runtime/lua-53
REQUIRED_PACKAGES += system/library
REQUIRED_PACKAGES += system/library/math
REQUIRED_PACKAGES += web/curl

Yours is missing imagemagick but instead depends on sdl.
With yours I get the following errors (and most probably our official build server, too):

/oibuild/oi-userland/components/games/freeciv/build/manifest-i386-freeciv.depend has unresolved dependency '
    depend type=require fmri=__TBD \
        pkg.debug.depend.file=libMagickWand-6.Q16.so.5 \
        pkg.debug.depend.reason=usr/bin/freeciv-server \
        pkg.debug.depend.type=elf \
        pkg.debug.depend.path=lib/64 \
        pkg.debug.depend.path=usr/gcc/7/lib/amd64 \
        pkg.debug.depend.path=usr/lib/64'.
/oibuild/oi-userland/components/games/freeciv/build/manifest-i386-freeciv.depend has unresolved dependency '
    depend type=require fmri=__TBD \
        pkg.debug.depend.file=libMagickWand-6.Q16.so.5 \
        pkg.debug.depend.reason=usr/bin/freeciv-manual \
        pkg.debug.depend.type=elf \
        pkg.debug.depend.path=lib/64 \
        pkg.debug.depend.path=usr/gcc/7/lib/amd64 \
        pkg.debug.depend.path=usr/lib/64'.
/oibuild/oi-userland/components/games/freeciv/build/manifest-i386-freeciv.depend has unresolved dependency '
    depend type=require fmri=__TBD \
        pkg.debug.depend.file=libMagickWand-6.Q16.so.5 \
        pkg.debug.depend.reason=usr/bin/freeciv-gtk3 \
        pkg.debug.depend.type=elf \
        pkg.debug.depend.path=lib/64 \
        pkg.debug.depend.path=usr/gcc/7/lib/amd64 \
        pkg.debug.depend.path=usr/lib/64'.
gmake: *** [/oibuild/oi-userland/make-rules/ips.mk:502: /oibuild/oi-userland/components/games/freeciv/build/.resolved-i386] Error 1

**Comment by AndWac** [comment]
Ok, looks good now. Please squash the commits and remove unnecessary comments before merge.

**Comment by AndWac** [comment]
Something is wrong with your PR as it contains commits from other people.

**Comment by jakegoerzen** [comment]
Sorry, I made a mistake trying to squash my commits.  I'll have to practice with git before submitting another PR.

**Comment by AndWac** [comment]
I had similar problems in the past. Typically this can be solved easily - you just need to rebase your branch on an updated oi/hipster, eg. like this:
git checkout oi/hipster
git pull upstream/oi/hipster   (update your local oi/hipster branch from our official repository)
git checkout freeciv
git rebase oi/hipster (put your commits "on top")
git rebase -i HEAD~<n>  (interactively rebase your branch in order to squash your <n> commits into one and cleaning the commit messages, <n> being a natural number)
git push -f origin freeciv (force push your local freeciv branch to your's on Github)

**Pull request closed without merging**